pkg/descheduler/strategies/lownodeutilization_test.go

```diff
@@ -103,7 +104,7 @@ func TestLowNodeUtilization(t *testing.T) {
 				},
 				n2NodeName: {
 					Items: []v1.Pod{
-						*test.BuildTestPod("p9", 400, 0, n1NodeName, test.SetRSOwnerRef),
+						*test.BuildTestPod("p9", 400, 0, n2NodeName, test.SetRSOwnerRef),
```

Comment:
Here and in the 3 following changes, I thought that was a typo as the pod is actually supposed to be on node 2. Not sure if that is either important or correct.

Reply:
Please put this changes into a separate PR.

Reply:
Already fixed on master by this [commit](https://github.com/kubernetes-sigs/descheduler/commit/81b816d4a4ff6417009cd7bd384d0da87b802b44#diff-67abe73ca76878708a926cd707d8d6a205acc3706a83802459b0a5082234168fL166).